CONTENTSTYLEGUIDE.md

```diff
@@ -23,19 +23,18 @@ Test names will use the syntax `test_mainVar_In/correct_*V#` to support brevity
 
 ```
 test_ExampleVar_Correct_V1 if {
-    ControlNumber := "Example 2.1"
-    Requirement := "Example Policy Bullet Point"
-    
+    PolicyId := "MS.<Product>.<Policy #>.<Bulletpoint #>v<Version #>"
```

Comment:
Might be worth having a quick discussion with the entire team as to naming conventions.  I wouuld have called it "MS.<Product>.<Control Group #>.<Control #>v<Version #>.  But this is biased from work Andrew and Aldin did at the hackathon on parsing the baseline documents.

Reply:
I don't have a strong opinion one way or another